os/android/drmhwctwo.h

```diff
@@ -239,6 +239,9 @@ class DrmHwcTwo : public hwc2_device_t {
   hwcomposer::GpuDevice device_;
   std::map<hwc2_display_t, HwcDisplay> displays_;
   std::map<HWC2::Callback, HwcCallback> callbacks_;
+
+ private:
+  static bool is_explicit_sync_enabled_;
```

Comment:
@kalyankondapally the explicit sync query is done in hwc initialization, when the common display objects, like render are not created yet. we cannot assign its member variables. The current static is a workaround. I prefer to let it in first. Idealy, we need refactoring the code, and use a singleton class to manage the drm feature and capability.